### PR TITLE
Add change log for Nov release

### PR DESCRIPTION
**Pull request by Vidya2606**
This PR is to create change log for November release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ If you are using one of the IDE extensions for Bridge to Kubernetes, check the r
 - [Visual Studio 2019 extension release notes](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.mindaro#whats-new)
 - [Visual Studio 2022 extension release notes](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.mindaro2022#whats-new)
 
+## [1.0.20231128.1]
```
Comment (hsubramanianaks) [inline CHANGELOG.md:10]: version should be 1.0.20231127.3 right ?

**Review by Tatsinnit — DISMISSED** [review]
👍❤️🎉 looks good to me, thank you so much for this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ If you are using one of the IDE extensions for Bridge to Kubernetes, check the r
 - [Visual Studio 2019 extension release notes](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.mindaro#whats-new)
 - [Visual Studio 2022 extension release notes](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.mindaro2022#whats-new)
 
+## [1.0.20231127.3]
+- [Support arm64 binaries for linux, windows and macos](https://github.com/Azure/Bridge-To-Kubernetes/pull/290)
+- [Update package version](https://github.com/Azure/Bridge-To-Kubernetes/pull/330)
+- [Update build-pipelines-PR.yml for Azure Pipelines](https://github.com/Azure/Bridge-To-Kubernetes/commit/65a0527df3ad85525668c05e8737de71247087ab)
```
Comment (hsubramanianaks) [inline CHANGELOG.md:13]: @Vidya2606 I merged a PR recently can you add them to changeLog please which is required for this release.

**Review by hsubramanianaks — APPROVED** [review]
looks good to me.

**Pull request merged**